packages/code-du-travail-frontend/src/modules/layout/footer/index.tsx

```diff
@@ -8,7 +8,7 @@ import { FooterBottom } from "./FooterBottom";
 
 export const Footer = () => {
   return (
-    <footer role="contentinfo">
+    <footer>
```

Comment:
j'ai enlevé ça ça faisait une erreur dans le test validate-html. J'ai suivi la recommandation [ici](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/contentinfo_role) "This is a website footer. Using the [<footer>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer) element instead is recommended:"